crypto/openssh/version.h

```diff
@@ -5,4 +5,4 @@
 #define SSH_PORTABLE	"p1"
 #define SSH_RELEASE	SSH_VERSION SSH_PORTABLE
 
-#define SSH_VERSION_FREEBSD	"FreeBSD-20240104"
+#define SSH_VERSION_FREEBSD	"CheriBSD-20240701"
```

Comment:
I'm not sure this is the best value as it's probably the wrong one for older releng branches which will also need patching. I think there's probably no good option

Reply:
I wonder if this will require patching some third-party software that detects a remote OS.

Reply:
The 14.0 patch bumps it to this version too, which seems ill-advised, but means we'd be consistent to use FreeBSD-...

Reply:
And the 13.3 patch bumps from this exact version: https://security.FreeBSD.org/patches/SA-24:04/openssh-13.3.patch

Reply:
I'll switch back to FreeBSD-